src/chapters/setup-intro-data-types/exercises.rst

```diff
@@ -1,2 +1,210 @@
+
+   .. figure:: figures/nameClass.png
+      :alt: Name Class
+
+      Name Class
+
+
+   .. note:: 
+      You will be asked whether you want to add this file to Git.
+      Press the “Yes” button.
+   
+   .. figure:: figures/addToGit.png
+      :alt: Add class to Git
+
+      Add class to Git
+
+#. The new class is now created! You can proceed to write code within
+   it. (Don’t forget to write the ``main`` method!)
+
+   .. figure:: figures/newClassCreated.png
+      :alt: Ready to start
+
+      Ready to start
+
+Exercises
+---------
+
+#. **Input/output**: Write a new “Hello, World” program to prompt the
+   user for their name and greet them by name.
+
+   #. Follow steps 3-5 above to create a new Class inside of your exercises.
+   #. Add an import statement at the the top of the file to include Scanner:
+
+      .. sourcecode:: java
+
+         import java.util.Scanner;
+
+   #. Declare a variable of type ``Scanner`` called ``in``:
+
+      .. sourcecode:: java
+
+         Scanner in = new Scanner(System.in);
+
+   #. Add a question to ask the user:
+
+      .. sourcecode:: java
+      
+         System.out.println("Hello, what is your name:");
+
+   #. Create a variable to store the user's response using the ``Scanner``'s ``.nextLine()`` method
+
+      .. sourcecode:: java
+
+         String name = in.nextLine();
+
+   #. Use concatenation to print the greeting:
+
+      .. sourcecode:: java
+
+         System.out.println("Hello " + name);
+   
+   #. Right-click/Control-click the arrow next to your class and run the program.
+
+
+#. **Numeric types**: Write a program to calculate the area of a
+   rectangle and print the answer to the console. You should prompt the
+   user for the dimensions. (What data types should the dimensions be?)
+   
+   #. Follow steps 3-5 above to create a new Class inside of your exercises.
+   #. Add an import statement at the top of your file to use ``Scanner``.
+   #. Add a ``Scanner`` object to handle the user's input.
+   #. Add a print line to prompt the user for length of the rectangle.
+   #. Define a variable to handle the user's response. Now is the time to know what type the 
+      dimension will be.
+   #. Repeat the previous two steps to ask for and store the rectangle width.
+   #. Use the length and width values to calculate the rectangle's area.
+   #. Print a statement using concatenation to communicate to the user what the area of 
+      their rectangle is. 
+   #. Run the program to verify your code.
+
+#. **Numeric types**: Write a program that asks a user for the number of
+   miles they have driven and the amount of gas they’ve consumed (in
+   gallons), and print their miles-per-gallon.
+#. **Strings**: The first sentence of *Alice’s Adventures in Wonderland*
+   is below. Store this sentence in a string, and then prompt the user
+   for a term to search for within this string. Print whether or not the
+   search term was found. See if you can make the search
```

Comment:
How about rephrasing to make the case-insensitivity mandatory instead of optional.  Something like, "Make the search case-insensitive..."